parcel_test.go

```diff
@@ -31,65 +33,117 @@ func getTestParcel() Parcel {
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite3", "tracker.db")
+	require.NoError(t, err)
+	defer db.Close()
+
 	store := NewParcelStore(db)
+
 	parcel := getTestParcel()
 
 	// add
 	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
+	parcel.Number, err = store.Add(parcel)
+
+	require.NoError(t, err)
+	require.NotEmpty(t, parcel.Number)
 
 	// get
 	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
 	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
+	parcelCheck, err := store.Get(parcel.Number)
+
+	require.NoError(t, err)
+	assert.Equal(t, parcel, parcelCheck)
 
 	// delete
 	// удалите добавленную посылку, убедитесь в отсутствии ошибки
+	err = store.Delete(parcel.Number)
+	require.NoError(t, err)
+
 	// проверьте, что посылку больше нельзя получить из БД
+	_, err = store.Get(parcel.Number)
+	require.Equal(t, sql.ErrNoRows, err)
```

Comment:
Лучше использовать специальные функции для ошибок
```suggestion
	require.ErrorIs(t, err, sql.ErrNoRows)
```